app/main.py

```diff
@@ -1,8 +1,82 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        if not (isinstance(comfort_class, int)
+                and comfort_class in range(1, 8)):
+            raise ValueError("comfort_class argument must be from 1 to 7")
+        self.comfort_class = comfort_class
+
+        if not (isinstance(clean_mark, int)
+                and clean_mark in range(1, 11)):
+            raise ValueError(
+                "clean_mark argument must be from very dirty - 1 to clean - 10"
+            )
+        self.clean_mark = clean_mark
+
+        if not (isinstance(brand, str)):
+            raise TypeError("brand argument is not string")
```

Comment:
It's a redundant check for the brand because your type annotation says about that. Remove in all similar cases